dev-libs/elftoolchain/elftoolchain-0.7.1.ebuild

```diff
@@ -0,0 +1,62 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit toolchain-funcs
+
+DESCRIPTION="Libraries/utilities to handle ELF objects (BSD drop in replacement for libelf)"
+HOMEPAGE="https://wiki.freebsd.org/LibElf"
+SRC_URI="https://netcologne.dl.sourceforge.net/project/elftoolchain/Sources/${P}/${P}.tar.bz2"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="
+	app-arch/libarchive
+	dev-libs/expat
+	dev-libs/libbsd
+	!dev-libs/elfutils
+	!dev-libs/libelf"
+DEPEND="${RDEPEND}
+	virtual/yacc"
+BDEPEND="
+	dev-vcs/subversion
+	sys-apps/lsb-release
+	sys-devel/bmake"
+
+# Requires network access and otherwise fails
+RESTRICT="test"
+
+src_prepare() {
+	default
+	rm -r test || die
+
+	sed -i -e "s@cc@$(tc-getCC)@" common/native-elf-format || die
+	sed -i -e "s@readelf@$(tc-getREADELF)@" common/native-elf-format || die
+}
+
+src_compile() {
+	export MAKESYSPATH="${EPREFIX}/usr/share/mk/bmake"
+	bmake \
+		AR="$(tc-getAR)" CC="$(tc-getCC)" \
+		LD="$(tc-getLD)" RANLIB="$(tc-getRANLIB)" || die
+}
+
+src_install() {
+	bmake \
+		DESTDIR="${D}" \
+		RANLIB="$(tc-getRANLIB)" \
+		-C libelf install || die
+
+	bmake \
+		DESTDIR="${D}" \
+		-C common install || die
+
+	find "${ED}" -name '*.la' -o -name '*.a' -delete || die
+
+	if [ "$(get_libdir)" = "lib64" ] ; then
```

Comment:
Bash tests please! It's too hard to get things right with POSIX tests and we're guaranteed to have Bash anyway (see PMS).